pkg/ebpf/ingress_node_firewall_loader.go

```diff
@@ -195,27 +198,30 @@ func (infc *IngNodeFwController) makeIngressFwRulesMap(ingFirewallConfig ingress
 }
 
 // IngressNodeFwAttach attach eBPF program to list interfaces and pin them to different pinDir
-func (infc *IngNodeFwController) IngressNodeFwAttach(ifacesName []string, isDelete bool) error {
+func (infc *IngNodeFwController) IngressNodeFwAttach(ifacesName []string, isDelete bool) ([]uint32, error) {
+	var ifIdlist = make([]uint32, 0)
+
 	objs := infc.objs
 	for _, ifaceName := range ifacesName {
 		ifaceName := ifaceName
 		// Look up the network interface by name.
 		iface, err := net.InterfaceByName(ifaceName)
 		if err != nil {
-			return fmt.Errorf("lookup network iface %q: %s", ifaceName, err)
+			return ifIdlist, fmt.Errorf("lookup network iface %q: %s", ifaceName, err)
 		}
+		ifIdlist = append(ifIdlist, uint32(iface.Index))
```

Comment:
can you return map[string]uint32  with ifaceName -> uint32? That way, callers of IngressNodeFwAttach can know which returned index belongs to which interface

Reply:
Will do